DzianCPP/monoshop
# 0003 client api (#10)

* controller to json brands

* api brand read-only controller

* api product read-only controller

* started unit tests

* simplified unit test for first ci/cd

* test workflow

* env

* learning correct ci/cd

* changes

* v of setup php action

* validate composer

* val composer

* name

* Compose json name property

* Composer description property

* Composer lock update

* Composer cache dir

* Install composer dep-s

* Check Symfony req-s

* Requirements for symfony

* Check symfony console

* php codestyle

* remove args

* remove php code style fixer

* run tests

* setup env and run tests workflow
